test/coldstorage/ColdStorage.t.sol

```diff
@@ -134,8 +139,18 @@ contract ColdStorageTest is RhinestoneModuleKit, Test {
 
         ERC7579BootstrapConfig[] memory validators =
             makeBootstrapConfig(address(ownableValidator), abi.encode(address(mainAccount.account)));
-        ERC7579BootstrapConfig[] memory executors =
-            makeBootstrapConfig(address(flashloanLender), abi.encode(""));
+
+        address[] memory addresses = new address[](2);
+        bytes[] memory callData = new bytes[](2);
+
+        addresses[0] = address(flashloanLender);
+        addresses[1] = address(coldStorageExecutor);
+
+        callData[0] = abi.encode("");
```

Comment:
just as a sidenote. ive been starting to use prb's solarray for this.

no need here, but just wanted to reference it for future use. makes things a bit more readable if the arrays are large